ui/lib/core/addon/components/page/error.hbs

```diff
@@ -7,25 +7,25 @@
   class="box is-sideless has-background-white-bis has-text-grey-light has-text-centered has-tall-padding"
   data-test-page-error
 >
-  {{#if (eq @error.httpStatus 404)}}
-    <h1 class="title is-3 has-text-grey-light" data-test-page-error-title={{@error.httpStatus}}>
+  {{#if (eq this.error.httpStatus 404)}}
+    <h1 class="title is-3 has-text-grey-light" data-test-page-error-title={{this.error.httpStatus}}>
       404 Not Found
     </h1>
-    <p>Sorry, we were unable to find any content at <code>{{@error.path}}</code>.</p>
-  {{else if (eq @error.httpStatus 403)}}
-    <h1 class="title is-3 has-text-grey-light" data-test-page-error-title={{@error.httpStatus}}>
+    <p data-test-page-error-subtitle>Sorry, we were unable to find any content at <code>{{this.error.path}}</code>.</p>
+  {{else if (eq this.error.httpStatus 403)}}
+    <h1 class="title is-3 has-text-grey-light" data-test-page-error-title={{this.error.httpStatus}}>
       Not authorized
     </h1>
-    <p>You are not authorized to access content at <code>{{@error.path}}</code>.</p>
+    <p data-test-page-error-subtitle>You are not authorized to access content at <code>{{this.error.path}}</code>.</p>
   {{else}}
-    <h1 class="title is-3 has-text-grey-light" data-test-page-error-title={{@error.httpStatus}}>
+    <h1 class="title is-3 has-text-grey-light" data-test-page-error-title={{this.error.httpStatus}}>
       Error
     </h1>
     <p>
-      {{#if @error.message}}
-        <p data-test-page-error-message>{{@error.message}}</p>
+      {{#if this.error.message}}
+        <p data-test-page-error-message>{{this.error.message}}</p>
```

Comment:
Could we use a div here instead of nesting a p tag inside another p tag?

Reply:
I changed the outer p to a div